src/main/kotlin/io/sip3/salto/ce/sip/SipTransaction.kt

```diff
@@ -22,11 +22,7 @@ import gov.nist.javax.sip.message.SIPResponse
 import io.sip3.salto.ce.Attributes
 import io.sip3.salto.ce.domain.Address
 import io.sip3.salto.ce.domain.Packet
-import io.sip3.salto.ce.util.callId
-import io.sip3.salto.ce.util.cseqMethod
-import io.sip3.salto.ce.util.fromUserOrNumber
-import io.sip3.salto.ce.util.toUserOrNumber
-import io.sip3.salto.ce.util.transactionId
+import io.sip3.salto.ce.util.*
```

Comment:
Not sure that usage of wildcard import is good idea for Kotlin extension function.